game/data/weapons.py

```diff
@@ -97,6 +97,7 @@ class WeaponType(Enum):
     ARM = "ARM"
     LGB = "LGB"
     TGP = "TGP"
+    DECOY = "DECOY"
```

Comment:
`DECOY = "decoy"`. The others are all caps because they're acronyms :)

Reply:
Ah good point, fixed :)